packages/network-controller/src/NetworkController.ts

```diff
@@ -654,6 +665,21 @@ export class NetworkController extends BaseController<
     };
   }
 
+  getSelectedNetworkClient():
```

Comment:
This method should have a JSDoc description.

Maybe also worth adding a `@deprecated` tag on `getProviderAndBlockTracker` (it's equivalent to this, except more difficult to work with because of the return type)

Reply:
done here: https://github.com/MetaMask/core/pull/4063/commits/af622c6e5d9f870c22c41142c519fe1fc2e9e070